conf.d/linux_proc_extras.yaml.example

```diff
@@ -0,0 +1,4 @@
+init_config:
+
+instances:
+  - tags: []
```

Comment:
Can you add some comment here to say that the configuration it doesn't need any specific parameters ?